main/python3/APKBUILD

```diff
@@ -27,7 +27,6 @@ prepare() {
 	cd "$builddir"
 	# force system libs
 	rm -r Modules/expat \
-		Modules/zlib \
```

Comment:
why that now needed? I mean it was removed but now we gonna compile it?

Reply:
zlib was removed from the Python source https://bugs.python.org/issue31358

Reply:
zlib was removed from the Python source https://github.com/python/cpython/pull/3375